tests/memory_tests/test_runner/requirements.txt

```diff
@@ -0,0 +1,8 @@
+pytest~=5.0
+attrs==19.1.0
```

Comment:
`attrs` was required for other `pytest` version to provide compatibility - shouldn't required for now and can be removed